fusor-ember-cli/app/templates/new-node-registration.hbs

```diff
@@ -1,117 +1,53 @@
-                                        type='text'
-                                        labelSize='col-xs-4'
-                                        inputSize='col-xs-6'
-                                        value=selectedNode.ipAddress
-                                        validator=ipAddressValidator
-                                        resetErrorsMessageKey=resetErrorsMessageKey
-                                        isRequired=true}}
-
-                                      {{text-f label="IPMI User"
-                                        type='text'
-                                        labelSize='col-xs-4'
-                                        inputSize='col-xs-6'
-                                        value=selectedNode.ipmiUsername
-                                        resetErrorsMessageKey=resetErrorsMessageKey
-                                        isRequired=false}}
-
-                                      {{text-f label="IPMI Password"
-                                        type='password'
-                                        labelSize='form-label-nowrap col-xs-4'
-                                        inputSize='col-xs-6'
-                                        value=selectedNode.ipmiPassword
-                                        resetErrorsMessageKey=resetErrorsMessageKey
-                                        isRequired=false}}
-
-                                  </fieldset>
-                                  <fieldset>
-                                      <legend>Networking</legend>
-                                      {{text-f label="NIC MAC Address"
-                                        type='text'
-                                        labelSize='form-label-nowrap col-xs-4'
-                                        inputSize='col-xs-6'
-                                        value=selectedNode.nicMacAddress
-                                        placeholder="unspecified"
-                                        validator=macAddressValidator
-                                        resetErrorsMessageKey=resetErrorsMessageKey
-                                        isRequired=true}}
-                                  </fieldset>
-                              </div>
-                          </div>
-                      </div>
-                </form>
-            </div>
-        {{/if}}
-    {{/em-modal-body}}
-
-    {{#em-modal-footer}}
-        <button type="cancel" class="btn btn-default" disabled={{async}} {{action "cancelRegisterNodes"}}>Cancel</button>
-        <button type="submit" class="btn btn-primary" disabled={{disableModalRegisterNodes}} {{action "registerNodes"}}>Register Nodes</button>
-    {{/em-modal-footer}}
-
-{{/em-modal}}
+{{#em-modal configName="bs" id="nodeRegistrationModal" class="node-registration-modal"
+            closeIfClickedOutside=handleOutsideClick open-if=openNewNodeRegistrationModal close-if=closeNewNodeRegistrationModal}}
```

Comment:
Re: `open-if` and `close-if`, I feel like I still haven't figured these out. Is `close-if` even required if `open-if` is defined? They seem redundant.

Also there's a `is-open` prop, to make things more confusing.